### ISSUE TITLE
'<' not supported between instances of 'int' and 'str'

### ISSUE THREAD

**Issue by TBeijen** [hellofresh/eks-rolling-update#26]
Ran into the error displayed below.

Command invoked:
```
GLOBAL_MAX_RETRY=30 python eks_rolling_update.py --cluster_name apps-1`
```

Kind of odd as recently a similar operation worked fine.
```
2019-12-12 06:54:58,337 INFO     Found 2 outdated instances
2019-12-12 06:54:59,462 INFO     Getting k8s nodes...
2019-12-12 06:55:00,790 INFO     Current k8s node count is 2
2019-12-12 06:55:00,790 INFO     Setting the scale of ASG apps-1-ng-a20191211120754245000000001 based on 2 outdated instances.
2019-12-12 06:55:00,791 INFO     Modifying asg apps-1-ng-a20191211120754245000000001 autoscaling to resume ...
2019-12-12 06:55:01,076 INFO     No previous capacity value tags set on ASG; setting tags.
2019-12-12 06:55:01,077 INFO     Saving tag to asg key: eks-rolling-update:original_capacity, value : 2...
2019-12-12 06:55:01,374 INFO     Saving tag to asg key: eks-rolling-update:desired_capacity, value : 4...
2019-12-12 06:55:01,601 INFO     Saving tag to asg key: eks-rolling-update:original_max_capacity, value : 5...
2019-12-12 06:55:01,933 INFO     Setting asg desired capacity from 2 to 4 and max size to 5...
2019-12-12 06:55:02,189 INFO     Waiting for 90 seconds for ASG to scale before validating cluster health...
2019-12-12 06:56:32,191 INFO     Describing autoscaling groups...
2019-12-12 06:56:33,206 INFO     Current asg instance count in cluster is: 4. K8s node count should match this number
2019-12-12 06:56:33,207 INFO     Checking asg apps-1-ng-a20191211120754245000000001 instance count...
2019-12-12 06:56:33,397 INFO     Asg apps-1-ng-a20191211120754245000000001 scaled OK
2019-12-12 06:56:33,398 INFO     Actual instances: 4 Desired instances: 4
2019-12-12 06:56:33,398 INFO     '<' not supported between instances of 'int' and 'str'
2019-12-12 06:56:33,398 INFO     *** Rolling update of asg has failed. Exiting ***
```

*Edit:* The `GLOBAL_MAX_RETRY` I use because of #22 . A quick scan through codebase suggests that's the culprit as it's parsed as `str`.

**Comment by chadlwilson** [comment]
This is a dupe of #23 which has a PR attached :)

**Comment by crhuber** [comment]
resolved  in #24
